kvmagent/kvmagent/plugins/tensorfusion/monitor.py

```
@@ -106,6 +106,18 @@ def clear(self, device_uuid):
             self._states.pop(device_uuid, None)
             self._notified_events.discard(device_uuid)
 
+    @staticmethod
+    def _is_vm_running(vm_uuid):
+        """Check if a VM is still running in libvirt."""
+        try:
+            from kvmagent.plugins.vm_plugin import LibvirtAutoReconnect
+            with LibvirtAutoReconnect() as conn:
+                dom = conn.lookupByName(vm_uuid)
+                return dom.isActive() == 1
+        except Exception:
```

Ruff (0.15.7)

[warning] Do not catch blind exception: `Exception`

(BLE001)